### PR TITLE
Added new options and changed Link of Home

### PR DESCRIPTION
**Pull request by fakhruddinkw**
**OLD**
mysite-old

**NEW**
mysite-new

Also in this PR, i have commented crypto link from navbar until we have crypto site up.

### CODE REVIEW

**Review by harshith-venkatesh — COMMENTED** [review]
Why are we having lint issue here

**Comment by fakhruddinkw** [comment]
> Why are we having lint issue here

I forgot to comment the crypto link at one place thats why

**Review by Aryex82 — APPROVED** [review]
LGTM

File: app/components/navbar.hbs [hunk 1]
```diff
@@ -1,35 +1,50 @@
...
+    <img class='user__pic' src={{@profilePictureURL}} alt='profile' />
+    <i class='fa-solid fa-chevron-down'></i>
   </div>
 
-  <section class="menu {{if this.isMenuOpen "active-menu" ""}}">
-    <LinkTo class="menu__link" @route="index">Home</LinkTo>
-    <LinkTo class="menu__link" @route="profile">Profile</LinkTo>
-    <LinkTo class="menu__link" @route="tasks">Tasks</LinkTo>
+  <section class='menu {{if this.isMenuOpen "active-menu" ""}}'>
+    <a href='https://www.realdevsquad.com' class='menu__link'>Home</a>
```
Comment (harshith-venkatesh) [inline app/components/navbar.hbs:35]: Would it be better to store link in a constant

**Comment by harshith-venkatesh** [comment]
Why are we moving to single quote here?

**Comment by harshith-venkatesh** [comment]
Screenshot 2023-08-08 at 12 31 42 AM

Given the above scenario and as the changes are minimal, link change and commenting of the crypto link, approving the PR

**Review by harshith-venkatesh — APPROVED** [review]

**Pull request merged**